File: src/Remove Element/Remove Element.py
class Solution:
	# @param    A       a list of integers
	# @param    elem    an integer, value need to be removed
	# @return an integer
	def removeElement(self, A, elem):
		index=0;
		for c in A:
			if c != elem:
				A[index] = c
				index+=1
		return index

File: src/Remove Element/Remove Element.cpp
class Solution {
	public:
		int removeElement(int A[], int n, int elem) {
			vector<int> temp(A, A+n);
			int index = 0;
			for(int i = 0 ; i < n ; i++)
			{
				if(temp[i] != elem) A[index++] = temp[i];
			}
			return index;
		}
};
